src/almond/worker_k8s.ts

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env node
+// -*- mode: typescript; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Almond
+//
+// Copyright 2021 The Board of Trustees of the Leland Stanford Junior University
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// load thingpedia to initialize the polyfill
+import 'thingpedia';
+
+import express from 'express';
+import WebSocket from "ws";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const wsjs = require('ws');
```

Comment:
Why do you need this?

Reply:
Please see my latest commit to get `transparent-rpc` to work over websocket.  
Let me know if it looks right.  I need to use createWebSockeStream which is not available in ts.